src/radical/pilot/agent/lm/__init__.py

```diff
@@ -3,5 +3,5 @@
 __license__   = "MIT"
 
 from .base import *
-from .base import LaunchMethod 
+from .base import LM
```

Comment:
Unsurprisingly, I liked the previous extended naming much better. What will be `LM` for a newbie? What will be for me in a week? As we say in Italian `Boh!`...

Reply:
I reviewed this again: the above was a consistency fix I stumbled over when renaming LRMS to RM.  Consistently renaming LM to LaunchMethod and RM to ResourceManager touches 55 files in about 250 places, and then some documentation.  I would thus like to leave that out of this PR and do that (with lower priority) in a separate refactoring.  If that works for you, would you mind opening a ticket?

Reply:
The LM/RM refactoring is done now.